test/xml/unit_point2grid.xml

```diff
@@ -219,4 +219,21 @@
       <grid_nc>&OUTPUT_DIR;/point2grid/point2grid_GOES_16_AOD_TO_G212.nc</grid_nc>
     </output>
   </test>
+
+  <test name="point2grid_NCCF_TO_G231">
```

Comment:
Howard, I notice that earlier in this file 2 tests are named the same thing:
```
TEST: point2grid_GOES_16_AOD_TO_G212_GAUSSIAN - pass -   1.628 sec
TEST: point2grid_GOES_16_AOD_TO_G212_GAUSSIAN - pass -   1.719 sec
```
Please modify one of their names to make them unique.

Reply:
I removed the duplicated unit test

Reply:
Howard, where did you remove the duplicate? It looks like both tests are still present on this current feature_1345 branch. Please be sure to push your local commits back up to GitHub so that they're included in the PR.